burst-gradle-plugin/src/test/projects/junit5/lib/build.gradle.kts

```diff
@@ -0,0 +1,8 @@
+plugins {
+  kotlin("jvm")
+  id("app.cash.burst")
+}
+
+dependencies {
+  testImplementation(kotlin("test"))
```

Comment:
I fail to see where junit5 is defined, is this the line adding the deps to junit5 instead of junit4?